skyportal/handlers/api/group.py

```diff
@@ -354,6 +354,22 @@ def post(self, group_id, *ignored_args):
                               type: boolean
                               description: Boolean indicating whether user is group admin
         """
+        current_groupuser = (
+            GroupUser.query.filter(GroupUser.group_id == group_id)
+            .filter(GroupUser.user_id == self.current_user.id)
+            .first()
+        )
+        if (
+            len(
+                {"System admin", "Manage users", "Manage groups"}.intersection(
+                    set(self.current_user.permissions)
+                )
+            )
+            == 0
+            and not current_groupuser.admin
```

Comment:
If a user is an admin, would they not also have some or all of the 3 ACLs above?

Reply:
No they would not necessarily -- a "Full user" will have none of these ACLs, but can still be an admin for a specific group